src/VkAuthAgent.php

```diff
@@ -114,7 +114,7 @@ private function retrieveRemixsid($cell, $password)
 
         preg_match('/name="ip_h" value="(.+)"/', $body, $matches);
         $ip_h = $matches[1];
-        preg_match('/name="lg_h" value="(.+)"/', $body, $matches);
+        preg_match('/name="lg_domain_h" value="(.+)"/', $body, $matches);
         $lg_h = $matches[1];
```

Comment:
Please rename the variable from `$lg_h` to `$lg_domain_h` and string occurancies as well.

Reply:
Done.